src/main/java/teammates/common/datatransfer/FeedbackSessionQuestionsBundle.java

```diff
@@ -26,6 +29,17 @@ public FeedbackSessionQuestionsBundle(FeedbackSessionAttributes feedbackSession,
         this.recipientList = recipientList;
     }
 
+    public FeedbackSessionQuestionsBundle(FeedbackSessionAttributes feedbackSession, Map<FeedbackQuestionAttributes,
+            List<FeedbackResponseAttributes>> questionResponseBundle, Map<String, Map<String, String>> recipientList,
+            Map<String, List<FeedbackResponseCommentAttributes>> commentsForResponses, CourseRoster roster) {
+
+        this.feedbackSession = feedbackSession;
+        this.questionResponseBundle = questionResponseBundle;
+        this.recipientList = recipientList;
+        this.commentsForResponses = commentsForResponses;
+        this.roster = roster;
```

Comment:
I think roster won't be null ever because this is how we are getting the roster ?
```
        CourseRoster roster = new CourseRoster(studentsLogic.getStudentsForCourse(courseId),
                instructorsLogic.getInstructorsForCourse(courseId));
```
Also if `EntityNotFoundException` is thrown, where should it be caught?

Reply:
The constructor above doesn't init roster.

Who throw `EntityNotFoundException`?

Reply:
I didn't understand. Why doesn't it init roster? When can be roster null?
> Who throw EntityNotFoundException?
I meant to say I was thinking of throwing this exception, but I dont understand how can roster be null when we are using the constructor mentioned above.

Reply:
```java
    public FeedbackSessionQuestionsBundle(FeedbackSessionAttributes feedbackSession,
            Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>> questionResponseBundle,
            Map<String, Map<String, String>> recipientList) {
        this.feedbackSession = feedbackSession;
        this.questionResponseBundle = questionResponseBundle;
        this.recipientList = recipientList;
    }
```

Here is the contructor, does roster get initilized?

In addition, you are going to use `Assumption.assertNotNull()` rather than throw exception.

Reply:
Okay, by the way this constructor isn't used anymore. And there are some more which are no longer in use because of changes in this PR. Can I remove them or removal of unused method be out out of scope of this PR?

Reply:
If the constructor is not used, why do we keep it?

Reply:
I dont know, I think it should be removed.